extensions/positron-assistant/src/openai-fetch-utils.ts

```diff
@@ -124,8 +127,42 @@ function transformServerSentEvents(text: string, providerName: string): string {
 				// Fix empty role fields in delta objects within choices array
 				if (data.choices && Array.isArray(data.choices)) {
```

Comment:
Using the type guard function from above:

```suggestion
				if (isChatCompletionChunk(data)) {
```

(Although, do Snowflake and others provide chunks that don't quite fit this format?)